components/cam/bld/configure

```diff
@@ -1527,21 +1496,12 @@ else {
             $nadv += 8;
             if ($print>=2) { print "Advected constituents added by $microphys_pkg microphysics: 8$eol"; }
         }
-        elsif ($microphys_pkg =~/^p3/) {
-            $nadv += 8;
-            if ($print>=2) { print "Advected constituents added by $microphys_pkg microphysics: 8$eol"; }
-        }
 
 	if ($clubb_do_adv) {
 	    $nadv += 9;
             if ($print>=2) { print "Advected constituents added by $microphys_pkg microphysics: 8$eol"; }
 	}
 
-        if ($shoc_sgs){
```

Comment:
just noting locations where shoc/p3 might be removed.

Reply:
ditto